update-vendor-firmware

```diff
@@ -57,7 +57,7 @@ if [ -e "$TARGET_MANIFEST" ]; then
 fi
 
 echo "Extracting updated vendor firmware..."
-tar xf "$VENDORFW/firmware.tar"
+tar -xf "$VENDORFW/firmware.tar" --overwrite
```

Comment:
Not every variant of `tar` will handle options at the end of the command. You should put `--overwrite` before the positional argument of the tarball filepath.

Reply:
Fixed, thanks

Reply:
Wanna mark this as approved @Conan-Kudo ? The --overwrite was moved :)